src/lib.rs

```diff
@@ -273,11 +273,18 @@ impl ErasureCoder {
     }
 
     /// Reconstructs the fragment specified by the given index from other available fragments.
+    /// The index must be lower than the total number of parity_fragments and data_fragments.
```

Comment:
In Rust, the `# Errors` section is conventionally used for describing conditions in which `Err(_)`s are returned (cf. https://doc.rust-lang.org/book/second-edition/ch14-02-publishing-to-crates-io.html#commonly-used-sections ).

Reply:
fixed

Reply:
Thanks!